packages/react-devtools-inline/src/frontend.js

```diff
@@ -58,25 +61,47 @@ export function createBridge(contentWindow: any, wall?: Wall): FrontendBridge {
 export function initialize(
   contentWindow: any,
   {
-    bridge,
-    store,
+    createBridge: customCreateBridge,
+    createStore: customCreateStore,
```

Comment:
Why do you need these custom functions?

Reply:
In our use case, I need to unload and reload the DevTools without refreshing the whole page. What I do in this PR is reloading the app iframe only, and destroy then create a new store, so the main page won't need to refresh.